### PR TITLE
feat: send message permission

### PR DESCRIPTION
**Pull request by minh-rakuten**
# Description
- Add send message permission to bridge.
- Request send message permission before sending message in sample app.

## Links
MINI-3193

# Checklist
- [x] I wrote/updated tests for new/changed code.
- [x] I ran `yarn sdk buildSdk` and `yarn bridge buildBridge` without issues.
- [x] I ran `yarn sample prettify` and `yarn sample build` without issues.

### CODE REVIEW

**Review by corycaywood — APPROVED** [review]
👍

File: js-miniapp-bridge/src/types/custom-permissions.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ export enum CustomPermissionName {
   CONTACT_LIST = 'rakuten.miniapp.user.CONTACT_LIST',
   ACCESS_TOKEN = 'rakuten.miniapp.user.ACCESS_TOKEN',
   LOCATION = 'rakuten.miniapp.device.LOCATION',
+  SEND_MESSAGE = 'rakuten.miniapp.action.SEND_MESSAGE',
```
Comment (rleojoseph) [inline js-miniapp-bridge/src/types/custom-permissions.ts:7]: @corycaywood Are we planning to use the same name in Android & iOS ?

As of now, in iOS it is `rakuten.miniapp.user.SEND_MESSAGE`, I was suggesting that we will change it to `rakuten.miniapp.user.action.SEND_MESSAGE`.

What do you think ?

Comment (corycaywood) [reply js-miniapp-bridge/src/types/custom-permissions.ts:7]: Yeah, it should be the same on both platforms. I don't really have an opinion on which one we should use. @minh-rakuten What do you think?

Comment (minh-rakuten) [reply js-miniapp-bridge/src/types/custom-permissions.ts:7]: We will use `rakuten.miniapp.user.action.SEND_MESSAGE`. cc @khairul-alam-licon

Comment (khairul-alam-licon) [reply js-miniapp-bridge/src/types/custom-permissions.ts:7]: Ok, I'll update Android using `rakuten.miniapp.user.action.SEND_MESSAGE`

**Review by khairul-alam-licon — APPROVED** [review]
👍 👍

**Pull request merged**